frontend/app/components/work-packages/wp-single-view/wp-single-view.directive.html

```diff
@@ -23,81 +52,64 @@
     </div>
   </div>
 
-  <div ng-repeat="group in $ctrl.groupedFields" ng-hide="$ctrl.shouldHideGroup(group.groupName)"
+  <div ng-repeat="group in $ctrl.groupedFields track by group.name" ng-hide="$ctrl.shouldHideGroup(group)"
+       data-group-name="{{ group.name }}"
        class="attributes-group">
 
     <div class="attributes-group--header">
       <div class="attributes-group--header-container">
         <h3 class="attributes-group--header-text"
-            ng-bind="$ctrl.I18n.t('js.work_packages.property_groups.' + group.groupName)"></h3>
-      </div>
-      <div class="attributes-group--header-toggle">
-        <panel-expander tabindex="-1" ng-if="$first"
-                        collapsed="$ctrl.hideEmptyFields"
-                        expand-text="{{ $ctrl.I18n.t('js.label_show_attributes') }}"
-                        collapse-text="{{ $ctrl.I18n.t('js.label_hide_attributes') }}">
-        </panel-expander>
+            ng-bind="group.name"></h3>
```

Comment:
The groups will always contain the translated name. There is no mapping from an internal group key (e.g., `:estimates_and_time`) to its localization, since there is no such respective feature for names the user enters.